internal/query/user_membership.go

```diff
@@ -335,15 +339,23 @@ func prepareOrgMember(withOwnerRemoved bool) (string, []interface{}) {
 		"NULL::TEXT AS "+membershipProjectID.name,
 		"NULL::TEXT AS "+membershipGrantID.name,
 	).From(orgMemberTable.identifier())
+
+	for _, q := range query.Queries {
+		if q.Col().table.name == membershipAlias.name || q.Col().table.name == memberTableAlias.name {
+			builder = q.toQuery(builder)
```

Comment:
i think we could also remove the memberTableAlias as the membershipAlias is an alias for that